.github/workflows/sdk.yml

```diff
@@ -206,9 +208,11 @@ jobs:
         run: sudo chmod 777 /usr/local/bin && make build
         working-directory: sdk
         env:
+          RAML_FILE: "../api/api-specs/api/api.raml"
           API_RAML: "../api/api-specs/api/api.raml"
```

Comment:
What is the difference between RAML_FILE and API_RAML?

Reply:
It’s a fallback or if you want just to build a specific SDK